gamemodes/terrortown/gamemode/client/cl_player_ext.lua

```diff
@@ -7,6 +7,14 @@ if not plymeta then
 	return
 end
 
+---
+-- @realm client
+local cvEnableBobbing = CreateConVar("ttt2_enable_bobbing", "1", FCVAR_ARCHIVE)
+
+---
+-- @realm client
+local cvEnableBobbingStrafe = CreateConVar("ttt2_enable_bobbing_strafe", "1", FCVAR_ARCHIVE)
+
```

Comment:
I'm not a fan of enabling these by default.

Having a setting, that is generally known to cause motion sickness for some people, turned on by default just doesn't seem like a good idea to me.

Reply:
I can't think of any game that has them disabled by default. In all games that I play you have to disable them in the accessability settings if you don't like it